src/Files.App/Commands/CommandCodes.cs

```diff
@@ -20,11 +20,17 @@ public enum CommandCodes
 		RestoreRecycleBin,
 		RestoreAllRecycleBin,
 
-    // Start
+		// selection
+		MultiSelect,
+		SelectAll,
+		InvertSelection,
+		ClearSelection,
+
+		// start
 		PinToStart,
 		UnpinFromStart,
 
-		// Favorites
+		// favorites
```

Comment:
Why this change?

Reply:
The other sections should be capital as well.

Reply:
It's done.